core/src/banking_stage.rs

```diff
@@ -637,6 +642,22 @@ impl BankingStage {
         threads
     }
 
+    fn spawn_internal_unified(&self) -> Vec<JoinHandle<()>> {
+        info!("Spawning internal unified scheduler");
+        if !self.toggle_internal_unified(true) {
+            error!("Spawning unified scheduler failed");
+        }
+        // All unified scheduler threads are managed by itself. So, return none here.
+        vec![]
+    }
```

Comment:
What happens if this spawn errors, do we just run without a scheduler or do we somehow fallback gracefully to greedy?